extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -529,14 +543,5 @@ private function getSitesApi() {
 
 }
 
-/**
- * used options:
- *
- * --first			-- run only once for first wiki in queue
- * --limit=<limit>	-- run for <limit> wikis
- */
-global $IP, $options;
-
-$wgAutoloadClasses[ "DumpsOnDemand" ] = "$IP/extensions/wikia/WikiFactory/Dumps/DumpsOnDemand.php";
```

Comment:
? what's that?

Reply:
isn't it needed anymore?

Reply:
Because this script was a custom one MediaWiki was not fully set up when the script was run. Now we include the full MediaWiki stack at top of the script. All classes will be autoloaded properly.

Reply:
```
// make Wikia\Logger\Loggable trait available at a run-time
require_once( __DIR__ . '/../../../../lib/composer/autoload.php' );
 require_once( __DIR__ . "/../../../../maintenance/commandLine.inc" );	require_once( __DIR__ . '/../../../../maintenance/Maintenance.php' );
```
provides that